### ISSUE TITLE
About multi-label

### ISSUE THREAD

**Issue by MuZiGuYue** [flyingpot/pytorch_deephash#6]
Hello, I found that this paper deals with  single-label datasets，do you know if this paper can process multi-label datasets?

**Comment by spoonnotfound** [comment]
Yes, you can use this loss torch.nn.MultiLabelSoftMarginLoss() to deal with multi-label datasets.

**Comment by MuZiGuYue** [comment]
Do you know how to use Tensorflow to implement this paper?

**Comment by spoonnotfound** [comment]
Sorry, I don't know much about Tensorflow. You can implement it yourself.

2018-06-25 15:29 GMT+08:00 MuZiGuYue <notifications@github.com>:

**Comment by MuZiGuYue** [comment]
OK, thank you.

**Comment by josianerodrigues** [comment]
Hi @flyingpot,

To use the loss function torch.nn.MultiLabelSoftMarginLoss()  do I need to change the network structure or just modify the number of labels in the network output?
